ci/requirements-py3.10-pandas1.5.3-pydantic1.10.11.txt

```diff
@@ -4,6 +4,8 @@
 #
 #    pip-compile --output-file=ci/requirements-py3.10-pandas1.5.3-pydantic1.10.11.txt requirements.in
 #
+--index-url https://filipe_oliveira@mckinsey.com:cmVmdGtuOjAxOjE3MjI0NTc0NjU6Y2VLV3pGakNlc1hpaWJBNG1GQTR4bDQwT2Jj@mckinsey.jfrog.io/artifactory/api/pypi/python/simple
```

Comment:
@filipeo2-mck , time to swap your jfrog access token ASAP!

Reply:
Done! I'll check why this happened

Reply:
Unfortunately, default behavior of `pip-compile`, that emits the entire `index-url` by default (!): https://github.com/jazzband/pip-tools/issues/1876